x/evm/keeper/state_transition.go

```diff
@@ -362,6 +362,16 @@ func (k *Keeper) ApplyMessageWithConfig(
 		}
 	}
 	evm = k.NewEVM(ctx, msg, cfg, stateDB)
+	defer func() {
+		if e := recover(); e != nil {
+			if evm.Context.Random == nil {
```

Comment:
it's always `nil`, this will translate all kinds of panics, I think we should target the specific panic place in the go-ethereum.

Reply:
yea, since go-ethereum ensure this opcode use after [merge](https://github.com/ethereum/go-ethereum/pull/29387), we need handle in here

Reply:
we don't need this anymore if we just let it return zero.